### ISSUE TITLE
Request for New Version  to Be Deployed to Artifactory

### ISSUE THREAD

**Issue by pete-proton** [DanielaSfregola/random-data-generator#106]
Hello,

The source code of [RandomDataGenerator](https://github.com/DanielaSfregola/random-data-generator/blob/master/js/src/main/scala/com/danielasfregola/randomdatagenerator/RandomDataGenerator.scala) removes the dependency on `WeakTypeTag`, however the latest 2.8 version [available](https://search.maven.org/artifact/com.danielasfregola/random-data-generator_2.11/2.8/jar) in maven repo still depends on it.
![image](https://user-images.githubusercontent.com/26833375/73910260-29469b00-48f2-11ea-8f95-807fc4fe2265.png)

Could a version without a dependency on `WeakTypeTag` be deployed to maven repo?

Thanks in advance.

**Comment by DanielaSfregola** [comment]
Hi @pete-proton,
I am not sure I fully understand your request. You can see in the code that you posted that `WeakTypeTag` is used to extract information on the type `T`:
```
val tpe = implicitly[WeakTypeTag[T]].tpe
```

The library is using `WeakTypeTag`, and I believe we cannot remove it without changing its behaviour.

What issue you are experiencing with `WeakTypeTag`?

Cheers,
D.

**Comment by pete-proton** [comment]
hi, @DanielaSfregola,

thank you for prompt response. I confused something, the [class](https://github.com/DanielaSfregola/random-data-generator/blob/master/js/src/main/scala/com/danielasfregola/randomdatagenerator/RandomDataGenerator.scala) I pointed initially is a JS thing, the genuine [one](https://github.com/DanielaSfregola/random-data-generator/blob/master/jvm/src/main/scala/com/danielasfregola/randomdatagenerator/RandomDataGenerator.scala) still has weak type reference.

nevertheless, I get your point & will close the ticket.

> What issue you are experiencing with WeakTypeTag?

Basically I need to generate data based on the `Class.forName("some.package.className")` & with weak type tag it seems to be a bit difficult. Or is there a way you are aware of maybe?
